enums/uncefactVelocityUnits.json

```diff
@@ -0,0 +1,12 @@
+{
+    "description": "A subset of units for velocity from UN/CEFACT trade facilitation recommendation 20. In order: metre per second, knot, kilometre per hour, foot per second, mile per hour (statue mile)",
+    "type": "string",
+    "enum": [
+        "MTS",
+        "KNT",
+        "KMH",
+        "FR",
+        "HM",
+        "Beaufort"
```

Comment:
According to [this](https://view.officeapps.live.com/op/view.aspx?src=https%3A%2F%2Funece.org%2Ffileadmin%2FDAM%2Funcefact%2Frecommendations%2Frec20%2Frec20_Rev13e_2017.xls&wdOrigin=BROWSELINK) (Line 1529) Beaufort should be `M19`. Do we want to adopt this?

![image](https://user-images.githubusercontent.com/3358790/187134599-95570c9c-063e-476f-ba70-640e23baa708.png)

```suggestion
    "description": "A subset of units for velocity from UN/CEFACT trade facilitation recommendation 20. In order: metre per second, knot, kilometre per hour, foot per second, mile per hour (statue mile), Beaufort",
    "type": "string",
    "enum": [
        "MTS",
        "KNT",
        "KMH",
        "FR",
        "HM",
        "M19"
```